src/core/tests/Test_AptitudePackageManager.py

```diff
@@ -63,7 +65,7 @@ def mock_os_path_isfile_raise_exception(self, file):
     def mock_get_security_updates_return_empty_list(self):
         return [], []
 
-    #endregion Mocks
+    # endregion Mocks
```

Comment:
?

Reply:
We generally have a space between # and the text. Changed it here since there were other changes made to this file